src/Logging/ILoggerFactory.php

```diff
@@ -4,8 +4,11 @@
 namespace Momento\Logging;
 
 use Psr\Log\LoggerInterface;
+use Psr\Log\LogLevel;
 
 interface ILoggerFactory
 {
-    public function getLogger(?string $name): LoggerInterface;
+    public function getLogger(?string $name=null): LoggerInterface;
+
+    public function getLogLevel(): string|null;
```

Comment:
i'm not sure you want to put this here.

i am not familiar with the lay of the land in PHP logging but in other languages, logging frameworks will have their own way to configure the level, out-of-band from our code.  some may not even have levels.  So you really want the level to be an implementation detail for the individual logger, not something that we force through the logger factory.  e.g. see: 

https://github.com/momentohq/client-sdk-go/blob/main/examples/logging-example/momento_logrus/logrus.go

Reply:
Totally right, that wasn't supposed to be there, thanks. Fixed in a0f0e7e3.